src/compiler.ts

```diff
@@ -38,3 +48,34 @@ export interface SourceLocation {
   start: [number, number];
   end: [number, number];
 }
+
+/**
+ * Temp. target to dialect conversion function till we update prql-js.
+ * @param target Compilation target string.
+ */
+function targetToDialect(target: string | undefined) {
+  switch (target) {
+    case 'BigQuery':
+      return prql.Dialect.BigQuery;
+    case 'ClickHouse':
+      return prql.Dialect.ClickHouse;
+    case 'DuckDb':
+      return prql.Dialect.DuckDb;
+    case 'Generic':
+      return prql.Dialect.Generic;
+    case 'Hive':
+      return prql.Dialect.Hive;
+    case 'MsSql':
+      return prql.Dialect.MsSql;
+    case 'MySql':
+      return prql.Dialect.MySql;
+    case 'PostgreSql':
+      return prql.Dialect.PostgreSql;
+    case 'SQLite':
+      return prql.Dialect.SQLite;
+    case 'Snowflake':
+      return prql.Dialect.Snowflake;
+    default:
+      return prql.Dialect.Ansi;
+  }
+}
```

Comment:
I've just changes CompileOptions to take target as string: https://github.com/PRQL/prql/blob/main/prql-js/tests/test_all.js#L66-L69

... and added `get_targets()` that returns `['sql.postgres', 'sql.mysql', ...]`

Reply:
... which will greatly simplify this function.

Reply:
that function is a temp. till prql lib is updated. I plan to remove it, lowercase and append `sql.` if you want to stick with it for the final v. with updated js lib.